The make install target fails when using opam 2

I'm using opam 2, and the make install target fails, I suspect due to the sandboxing feature. I'm not sure how to solve this exactly, but the other mathcomp packages installed fine, so...

<details><summary>The log</summary>

```
 make -f Makefile.coq install
make[1]: Entering directory '/home/armael/.opam/current/.opam-switch/build/coq-mathcomp-analysis.dev'
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot change permissions of '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis/': No such file or directory
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
install: cannot create directory '/home/armael/.opam/current/lib/coq/user-contrib/mathcomp/analysis': Read-only file system
make[1]: *** [Makefile.coq:471: install] Error 1
make[1]: Leaving directory '/home/armael/.opam/current/.opam-switch/build/coq-mathcomp-analysis.dev'
``` 
</details>

--- a/opam
+++ b/opam
@@ -13,6 +13,8 @@ authors: [
 build: [
   [make "INSTMODE=global" "config"]
   [make "-j%{jobs}%"]
+]
+install: [
   [make "install"]
 ]
 remove: ["rm" "-R" "%{lib}%/coq/user-contrib/mathcomp/analysis"]

